config/src/parameters/actual.rs

```diff
@@ -90,6 +91,31 @@ pub struct Network {
     pub idle_timeout: Duration,
 }
 
+/// Wrapper around `iroha_genesis::GenesisSignature` for type-checking
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub struct GenesisSignatureConfig(iroha_genesis::GenesisSignature);
```

Comment:
It is a newtype needed to handle parsing of `GenesisSignature` from the user-provided format. It's place more in the `parameters::user` IMO.